### PR TITLE
monitoring: document slurm-exporter

### PR DESCRIPTION
**Pull request by heitorPB**
For our new charm: https://github.com/omnivector-solutions/charm-slurm-exporter/

Depends on https://github.com/omnivector-solutions/slurm-bundles/pull/24

Online at: https://omnivector-solutions.github.io/osd-documentation/testing/operations/monitoring.html

### CODE REVIEW

File: operations/monitoring.rst [hunk 1]
```diff
@@ -5,18 +5,21 @@ Monitoring
...
+`Prometheus <https://prometheus.io/>`_,
+`Prometheus node_exporter <https://github.com/prometheus/node_exporter>`_ and
+`slurm-exporter <https://charmhub.io/slurm-exporter>`_ to monitor the cluster
+and each individual node.
 
 
 Prometheus node_exporter
 ========================
 
+"in all nodes of the cluster"
```
Comment (dusktreader) [inline operations/monitoring.rst:17]: I don't understand what the context of this quote is.

File: operations/monitoring.rst [hunk 2]
```diff
@@ -35,6 +38,45 @@ Deploy Prometheus and relate it to node exporter to access this functionality:
...
 
 Please refer to these charm's documentation for configuration details.
+
+
+Prometheus Slurm exporter
+=========================
+
+The subordinate charm `slurm-exporter
+<https://charmhub.io/slurm-exporter>`_ exports metrics about Slurm:
+state of nodes, jobs, partitions, accounts, scheduler, CPUs, and GPUs. To
```
Comment (dusktreader) [inline operations/monitoring.rst:51]:
```suggestion
<https://charmhub.io/slurm-exporter>`_ exports metrics about Slurm such as
the state of nodes, jobs, partitions, accounts, scheduler, CPUs, and GPUs. To
```

File: operations/monitoring.rst [hunk 3]
```diff
@@ -35,6 +38,45 @@ Deploy Prometheus and relate it to node exporter to access this functionality:
...
+<https://charmhub.io/prometheus2>`_ charm to automatically scrape its metrics.
+Deploy Prometheus and relate it to ``slurm-exporter`` to access this
+functionality:
+
+.. code-block:: bash
+
+   $ juju deploy prometheus2
+   $ juju relate prometheus-node-exporter:prometheus prometheus2:scrape
+
+Please refer to these charm's documentation for configuration details.
```
Comment (dusktreader) [inline operations/monitoring.rst:78]:
```suggestion
Please refer to these charms' documentation for configuration details.
```

File: operations/monitoring.rst [hunk 4]
```diff
@@ -35,6 +38,45 @@ Deploy Prometheus and relate it to node exporter to access this functionality:
...
+
+You can use the `Grafana Dashboard 4323
+<https://grafana.com/dashboards/4323>`_.to visualize the metrics exported via
```
Comment (dusktreader) [inline operations/monitoring.rst:81]:
```suggestion
<https://grafana.com/dashboards/4323>`_ to visualize the metrics exported via
```

**Review by dusktreader — CHANGES_REQUESTED** [review]
Nice work! Left a few comments where I think the phrasing could be improved.

File: operations/monitoring.rst (hunk 1 above)
Comment (heitorPB) [reply operations/monitoring.rst:17]: Ouch, leftover from my notes. :facepalm:

**Comment by heitorPB** [comment]
Thanks for the feedback, @dusktreader! I addressed your comments.

**Review by dusktreader — APPROVED** [review]

**Pull request merged**